home/tests/test_api.py

```diff
@@ -655,6 +655,7 @@ def test_detail_view_whatsapp_message(self, uclient: Any) -> None:
         assert body["total_messages"] == 1
         assert body["text"]["type"] == "Whatsapp_Message"
         assert body["text"]["value"]["message"] == "*Default whatsapp Content 1* 🏥"
+        assert body["text"]["value"].get("title", "") == ""
```

Comment:
If we expect this field to always be in the response, we should index it directly like we do with the other fields.